cmake/developer_package/compile_flags/sanitizer.cmake

```diff
@@ -14,7 +14,7 @@ if (ENABLE_SANITIZER)
     set(SANITIZER_LINKER_FLAGS "${SANITIZER_LINKER_FLAGS} -fsanitize=address")
```

Comment:
For WIN32 please use VS syntax `/fsanitize=address`. Also, since many developer are not upgrading VS it would be nice if we run `check_cxx_compiler_flag("/fsanitize=address", SANITIZE_ADDRESS_SUPPORTED)` and print a human readable error that the compiler does not support an address sanitizer and point them on a wiki with pre-requisites https://github.com/openvinotoolkit/openvino/wiki/AddressSanitizer-and-LeakSanitizer

Reply:
Added in last commit